lua/lvim/utils/git.lua

```diff
@@ -78,12 +80,50 @@ function M.update_base_lvim()
     return
   end
 
+  -- Get the messages before merging
```

Comment:
please move these parts into `lua/lvim/core/info.lua`, since it might be possible to re-use some existing functions there.

Reply:
yes you are right this code should be there, I'll more it